ha/k8s_setup/ha_setup.py

```diff
@@ -274,6 +284,36 @@ def process(self):
         except Exception as c_err:
             sys.stderr.write(f'HA config command failed: {c_err}.\n')
 
+    def _add_node_health(self) -> None:
+        """
+        Add node health
+        """
+        confStoreAPI = ConftStoreSearch()
+        _, nodes_list = confStoreAPI.get_cluster_cardinality()
+        for node in nodes_list:
+            timestamp = str(int(time.time()))
```

Comment:
For now its ok but later I think we can add new function to just get return header from the utils Event class

Reply:
Sure.